tests/common.go

```diff
@@ -5315,6 +5315,217 @@ func GetSubsetOfSlice[T any](items []T, length int) ([]T, error) {
 	return randomItems, nil
 }
 
+// WaitTillPoolState returns true if the pool state matches the expected state
+func WaitTillPoolState(state opsapi.StorageRebalanceJobState) (bool, error) {
+	rebalanceFunc := func() (interface{}, bool, error) {
```

Comment:
please have a look at this function as the rebalance jobs will be created when explicitly run rebalance command